packages/svelte/src/compiler/phases/3-transform/client/transform-client.js

```diff
@@ -276,6 +276,13 @@ export function client_component(analysis, options) {
 			}
 		}
 
+		if (binding?.kind === 'prop' || binding?.kind === 'bindable_prop') {
+			return [
+				getter,
+				b.set(alias ?? name, [b.stmt(b.call(name, b.call('$.proxy', b.id('$$value'))))])
```

Comment:
The prop itself will be able to decide whether to proxy or not
```suggestion
				b.set(alias ?? name, [b.stmt(b.call(name, b.id('$$value')))])
```

Would also be good to have a small test for this

Reply:
Uh you are right...and i'm an idiot because i actually thought of adding a test and then forgot. Adding one rn.